src/stylesheets/colors.less

```diff
@@ -2,3 +2,4 @@
 @color_primary_light: @color_primary + #111;
 @color_primary_dark: @color_primary + #000;
 @color_text: #576988;
+@color_text_light: #C2C2C2;
```

Comment:
lets rename colors.less to constants.less, import at the top of main.less and then we shouldnt have to import constants.less at the top of inputs.less, for example